crates/taker-electron/src/main/main.ts

```diff
@@ -127,9 +166,11 @@ const createWindow = async () => {
  */
 
 app.on("window-all-closed", () => {
-    // Respect the OSX convention of having the application in memory even
-    // after all windows have been closed
-    if (process.platform !== "darwin") {
+    if (process.platform === "win32" && mainWindow) {
+        mainWindow.setSkipTaskbar(true);
+    } else if (process.platform === "darwin") {
+        app.dock.hide();
```

Comment:
is this one needed? if all windows are closed, hide should have no effect?

Reply:
This will hide the `dock` icon when closing the window, i.e. the icon in the app dock normally at the lower end of your screen.

Reply:
are you sure we want this behaviour? it's unexpected, in the sense that other apps don't behave like that when their window is hidden on macos.
There's typically another switch in the settings that allows the user to remove the app from the dock.

Reply:
I think it's good practice ? 🤷‍♂️
It was actually feedback from @holzeis [here](https://github.com/itchysats/itchysats/pull/3069#discussion_r984554809).

Reply:
interesting; from the UX perspective you typically want to go for "principle of least surprise", meaning that the app should behave the same way as all the other apps on the system.
I tested randomly some apps, and they do not disappear from dock on being hidden.
The closest example is probably Telegram, which has exactly the same options as us in the tray - and still does not remove the app from the dock.